slugg.js

```diff
@@ -6,8 +6,8 @@ var defaultToLowerCase = true
 
 function slugg(string, separator, toStrip) {
   // Ensure the given parameter is a string.
-  if (!(typeof string === 'string' || string instanceof String)) {
-    string = String((string == null) ? '' : string)
+  if (!(typeof string === 'string' || string instanceof String || Object.prototype.toString.call(string) === "[object String]")) {
```

Comment:
This if statement is pretty ugly, but wasn't sure of a cleaner way to detect a non-string.